app/models/developer_user.rb

```diff
@@ -0,0 +1,5 @@
+class DeveloperUser < ApplicationRecord
+    validates :public_key, presence: true
```

Comment:
CI is whining about using consistent spacing for this file, just back this spacing off to 2 spaces and it should be good.

Reply:
☝️ also, might be good to change the default on your editor